BootloaderCorePkg/Library/PciEnumerationLib/InternalPciEnumerationLib.h

```diff
@@ -101,6 +108,11 @@ struct _PCI_IO_DEVICE {
   //
   PCI_BAR                                   PciBar[PCI_MAX_BAR];
 
+  //
+  // Bus number ranges for a PCI Root Bridge device
+  //
+  PCI_BUS_NUM_RANGE                         *BusNumberRanges;

```

Comment:
Is this an array ?   if it is not,  is it better to use PCI_BUS_NUM_RANGE instead of PCI_BUS_NUM_RANGE*  ?   It will avoid dynamic allocation.

Reply:
Addressed.